src/common/QskArcBorderColors.h

```diff
@@ -0,0 +1,90 @@
+/******************************************************************************
+ * QSkinny - Copyright (C) 2016 Uwe Rathmann
+ * This file may be used under the terms of the QSkinny License, Version 1.0
+ *****************************************************************************/
+
+#ifndef QSK_ARC_BORDER_COLORS_H
+#define QSK_ARC_BORDER_COLORS_H
+
+#include "QskNamespace.h"
+
+#include <qcolor.h>
+#include <qmetatype.h>
+
+class QDebug;
+
+class QSK_EXPORT QskArcBorderColors
+{
+  public:
+    QskArcBorderColors();
+
+    QskArcBorderColors( const QColor& outer, const QColor& inner );
```

Comment:
We could actually make this class a pair of gradients, then we could draw conical gradients with the QPainter API